packages/shared/src/domAttrConfig.ts

```diff
@@ -63,6 +63,7 @@ export const isNoUnitNumericStyleProp = /*#__PURE__*/ makeMap(
     `grid-row,grid-row-end,grid-row-span,grid-row-start,grid-column,` +
     `grid-column-end,grid-column-span,grid-column-start,font-weight,line-clamp,` +
     `line-height,opacity,order,orphans,tab-size,widows,z-index,zoom,` +
+    `-webkit-line-clamp,` +
```

Comment:
Only adding this doesn't make sense, since every property here are non-prefixed standard properties. There are most likely other prefixed versions of some other properties that can be treated as numbers, and it is impractical for us to support all of them.

I think just supporting custom properties is good enough in this PR. Whether we want to support vendor-prefixed properties is a separate problem.

Reply:
Ok, I'll remove it.